src/librustdoc/html/static/search.js

```diff
@@ -1288,17 +1294,19 @@ window.initSearch = function(rawSearchIndex) {
     function buildIndex(rawSearchIndex) {
         searchIndex = [];
         var searchWords = [];
-        var i, word;
+        var i, word, normalizedName;
```

Comment:
Similar to above: I would prefer not to widen the scope of `normalizedName` unnecessarily. Instead, maybe rename either the outer `normalizedName` or the inner one?

Reply:
Actually, there is no need for the variable, I'll simply remove it.